cmd/govim/testdata/scenario_default/go_to_def_upper_case.txt

```diff
@@ -28,7 +28,10 @@ module mod.com
 
 go 1.12
 
-require example.com/blah v1.0.0
+require example.com/blaH v1.0.0
```

Comment:
I think this change is a bug fix, I'm I right? It just accidently worked before because `go.mod` got updated automatically to include both `blah` and `blaH` when updates to `go.mod` was allowed.